docker/Dockerfile

```diff
@@ -44,9 +44,10 @@ RUN \
     # Install rapidjson from source
     curl -kOL https://github.com/miloyip/rapidjson/archive/v1.0.2.tar.gz && \
     tar xfz v1.0.2.tar.gz && \
-    mv rapidjson-1.0.2/include/rapidjson/ /usr/local/include && \
+    mv rapidjson-1.0.2/include/rapidjson/ /usr/local/include
+RUN \
     # Install orion from source
-    git clone https://github.com/telefonicaid/fiware-orion && \
+    git clone https://github.com/giovannicuriel/fiware-orion && \
```

Comment:
Again, I'm not sure if this would be eligible for merging back at the original repo.

That said - I don't like how this container is built. It tends to take forever to build the image, and is prone to having unneeded packages installed on the resulting system (that is, container). I'm not sure whether this has come up as a discussion point there, but here (by here, please read the repo to be made "public") I think we might want to respectfully diverge.